modules/dnn/src/layers/pooling_layer.cpp

```diff
@@ -122,8 +122,17 @@ class PoolingLayerImpl CV_FINAL : public PoolingLayer
         }
         else
             CV_Error(Error::StsBadArg, "Cannot determine pooling type");
+
         setParamsFrom(params);
         ceilMode = params.get<bool>("ceil_mode", true);
+        if (params.has("is_global_pooling"))
```

Comment:
There is already "global_pooling" flag so let's just use it in the same way as we use vector of kernel sizes or strides.